frontend/src/views/DashboardManager/EmptyProgression.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <v-card outlined class="pa-4">
+    <v-row>
+      <v-col cols="12" sm="6" md="5" class="d-flex flex-column">
+        <div v-for="measure in keyMeasures" :key="measure.id" class="d-flex my-3">
+          <div class="text-body-1 font-weight-bold">
+            <v-icon class="mr-2" size="20" :color="measure.mdiIconColor">{{ measure.mdiIcon }}</v-icon>
+            {{ measure.shortTitle }}
+          </div>
```

Comment:
Maybe I'm being too picky in this code review seeing as it is just a placeholder, but can you use a p tag instead of a div ? Not sure if the html will be rejected with the v-icon inside though

Reply:
Fix : https://github.com/betagouv/ma-cantine/pull/2644/commits/9a1d37479c77c826dbfb8c9701fe07b9d91a125b